.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Please note that pull requests without a rationale and clear improvement are
```

Comment:
Please note that 'please note that' is conent-free in almost all circumstances, and is usually better left out. :)

Reply:
I'd also change 'are' to 'may be'